polars/polars-ops/src/chunked_array/strings/namespace.rs

```diff
@@ -63,11 +63,37 @@ pub trait Utf8NameSpaceImpl: AsUtf8 {
 
     #[cfg(feature = "string_from_radix")]
     // Parse a string number with base _radix_ into a decimal (i32)
-    fn parse_int(&self, radix: Option<u32>) -> Int32Chunked {
+    fn parse_int(&self, radix: u32, strict: bool) -> PolarsResult<Int32Chunked> {
+        use polars_arrow::utils::CustomIterTools;
         let ca = self.as_utf8();
+        let f = |opt_s: Option<&str>| -> Option<i32> {
+            opt_s.and_then(|s| <i32 as Num>::from_str_radix(s, radix).ok())
+        };
+        let out: Int32Chunked = ca.into_iter().map(f).collect_trusted();
+
+        if strict && ca.null_count() != out.null_count() {
+            let failure_mask = !ca.is_null() & out.is_null();
+            let failures = ca.filter(&failure_mask)?.unique()?;
```

Comment:
I was missing a unique_stable() method for ChunkedArrays to improve the err msg, I guess it could be implemented analogous to arg_unique in another PR